### PR TITLE
WIP: DIG-241 | RSS fields, data

### PR DESCRIPTION
**Pull request by dj1999**

### CODE REVIEW

File: config/sync/views.view.content_rss_lists.yml [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+        - 'languages:language_interface'
+        - url
+        - 'user.node_grants:view'
+        - user.permissions
+      tags:
+        - 'config:field.storage.node.field_excerpt'
+        - 'config:field.storage.node.field_featured_image'
+  feed_1:
+    display_plugin: feed
+    id: feed_1
```
Comment (szatom) [inline config/sync/views.view.content_rss_lists.yml]: Pls rename it to 'feed'

File: web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+use Drupal\node\Plugin\views\row\Rss;
+use Drupal\node\Entity\Node;
+
+/**
+ * Plugin for resulting object and formats it as an RSS item.
+ *
+ * @ViewsRow(
+ *   id = "diginomica_rss",
+ *   title = @Translation("Diginomica Content"),
+ *   help = @Translation("Display the Diginomica content with standard node view."),
```
Comment (szatom) [inline web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php]: ?

File: web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+   * Prepare category items for rss.
+   *
+   * @param \Drupal\node\Entity\Node $entity
+   *   The node being create rss.
+   * @param array $elements
+   *   The elements array.
+   */
+  protected function getCategories(Node $entity,
+    array &$elements) {
+    foreach ($entity->field_category->referencedEntities() as $category) {
```
Comment (szatom) [inline web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php]: pls check: hasField('field_category')

File: web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php (hunk 2 above)
Comment (dj1999) [reply web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php]: I don't understand.

File: config/sync/views.view.content_rss_lists.yml (hunk 1 above)
Comment (dj1999) [reply config/sync/views.view.content_rss_lists.yml]: Renamed but first have to rename author feed ftom feed -> feed_author

File: web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php (hunk 2 above)
Comment (szatom) [reply web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php]: My question mark is assigned to the description, I think it's not about 'content with standard node view.', it is rss

**Comment by szatom** [comment]
Se comment in the JIRA issue.

File: web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php (hunk 2 above)
Comment (dj1999) [reply web/modules/custom/diginomica/src/Plugin/views/row/DiginomicaRss.php]: Removed this plugin wrote custom controller.

**Pull request merged**